internal/config/config.go

```diff
@@ -87,17 +89,20 @@ func userAgent(terraformVersion *string) string {
 
 // authenticationOption computes the desired authentication option for the *management.Management client.
 func authenticationOption(clientID, clientSecret, apiToken, audience string) management.Option {
+	ctx := context.Background()
```

Comment:
This seems to be needed as otherwise we're gonna have the context canceled after a few API calls as the CRUD funcs will do a ctx.Done() call.